source/utils/CMakeLists.txt

```
@@ -12,6 +12,9 @@ configure_file(
 # BPLS
 add_executable(bpls ./bpls/bpls.cpp)
 target_link_libraries(bpls adios2_core adios2sys adios2::thirdparty::pugixml)
+if(WIN32)
+  target_link_libraries(bpls shlwapi)
+endif()
```

Actually this might be even better. I do not think that pugixml needs to be a  public dep.

```suggestion
target_link_libraries(bpls
                      PUBLIC adios2_core adios2sys 
                      PRIVATE adios2::thirdparty::pugixml $<$<PLATFORM_ID:Windows>:shlwapi>)
```

---

Done.